docs/grafana.rst

```diff
@@ -0,0 +1,320 @@
+Real time graphs
+====================
+
+Grafana
+-----------
+
+`Grafana <https://grafana.com/>`_ is the open source analytics and monitoring solution and may be used to view various BEAM metrics in real time during BEAM execution.
+
+Grafana is a tool to display data from data source. In our case the data source is `InfluxDB <https://www.influxdata.com/products/influxdb-overview/>`_ .
+
+
+Docker installation and configuration
+--------------------------------------
+
+There is a docker image with configured Grafana and InfluxDB to work with BEAM thus one need to install docker in order to use Grafana.
+
+Ubuntu Linux users
+^^^^^^^^^^^^^^^^^^^
+
+Requirements
+"""""""""""""""""""
+
+* 64-bit version of one of these Ubuntu versions: Disco 19.04, Cosmic 18.10, Bionic 18.04 (LTS), Xenial 16.04 (LTS)
+
+Installation
+"""""""""""""""""""
+
+There is an official `installation guide for Ubuntu Linux users <https://docs.docker.com/install/linux/docker-ce/ubuntu/>`_ and
+a `Post-installation guide for Linux <https://docs.docker.com/install/linux/linux-postinstall/>`_ with optional procedures
+for configuring Linux hosts to work better with Docker for non-root users.
+
+Otherwise use those commands::
+
+    sudo apt-get update
+    sudo apt-get install docker-ce docker-ce-cli containerd.io docker-compose
+
+
+Mac users
+^^^^^^^^^^^^
+
+Requirements
+"""""""""""""""""""
+
+* Mac hardware must be a 2010 or newer model
+* macOS must be version 10.13 or newer
+* VirtualBox prior to version 4.3.30 must not be installed as it is not compatible with Docker Desktop.
```

Comment:
Is this a grammar issue or would having an old VirtualBox be problematic? Do you mean to say that it must be newer than 4.3.30?

Reply:
It is copupasted directly from official instruction -> https://docs.docker.com/docker-for-mac/install/
It seems, that old virtualbox is not compatible with docker.

Reply:
I still find it weird grammatically, but am fine with that